troposphere/static/js/components/images/detail/versions/Version.react.js

```diff
@@ -108,29 +92,55 @@ export default React.createClass({
 
     },
 
-    renderDetail() {
+    renderChangeLog() {
         let { version } = this.props;
         let styles = this.styles();
-
         let changeLog = version.get("change_log");
         let converter = new showdown.Converter();
         let changeLogHTML = converter.makeHtml(changeLog);
 
         return (
-            <div
-                style={ styles.content }
-            >
-                <div style={ styles.description }
-                    dangerouslySetInnerHTML={{
-                        __html: changeLogHTML
-                    }}
-                />
-                <div style={ styles.availability }
-                >
-                    { this.renderAvailability() }
-                </div>
+        <div style={ styles.description }
+            dangerouslySetInnerHTML={{
+                __html: changeLogHTML
+            }}
+        />
+       );
+    },
+
+    renderSummary() {
```

Comment:
We want to also render the providers for the summary. Just with the `isSummary` prop so the `uuid`s are hidden.